helper/benchhelpers/benchhelpers.go

```diff
@@ -0,0 +1,19 @@
+package benchhelpers
+
+import (
+	"testing"
+
+	testing2 "github.com/mitchellh/go-testing-interface"
```

Comment:
```suggestion
	testinginterface "github.com/mitchellh/go-testing-interface"
```

Reply:
:+1: I initially had something like this, but GoLand "helpfully" changed it when I refactored something. I'll switch it back.